Hai Reveal conversations: dialog improvements to A08 Symbol Identification (#7581)
endless-sky/endless-sky: data/hai/hai reveal 4 middle.txt

		log "Giti identified the angry cat symbol. It is from a long-defunct subversive group within the Syndicate called Base Principles. Somehow they obtained alien technology, and used the knowledge gained from it to advance their influence within the organization. A half-century ago, Syndicate Internal Affairs scoured them from the Syndicate's ranks in an extremely thorough and prejudiced inquisition."
		
		conversation

> 			`You land at Ulaanbaatar's spaceport and receive a message that Batu has to finalize some arrangements and will arrive in the evening. That leaves plenty of time for you to see to your business of angry cat symbols.`
> 			`	At an information desk, you inquire about the route to the library, and are given two options. You can take a very expensive private taxi service, which will be payed for by the Syndicate. Or, you can take the long route and see the sights. Either way, you'll have plenty of time before Batu arrives.`

Reviewer: ```suggestion
			`You land at Ulaanbaatar's spaceport and receive a message that Batu has to finalize some arrangements and will arrive in the evening. That leaves plenty of time for you to see to your other business about the symbol.`
			`	At an information desk, you inquire about the route to Moscow, and are given two options. You can take a very expensive private taxi service, which you could bill to Alondo. Or, you can take the long route and see the sights. Either way, you'll have plenty of time before Batu arrives.`
```

> 			`	At an information desk, you inquire the route to the library, and there are two options. You can take a - very expensive - private hover-taxi service, which will be payed for by the Syndicate. Or, you can take the long route and see the sights. Either way, you'll have plenty of time before Batu arrives`

Reviewer 2: ```suggestion
			`	At an information desk, you inquire about the route to the library, and are given two options. You can take a very expensive private hovertaxi service, which will be payed for by the Syndicate. Or, you can take the long route and see the sights. Either way, you'll have plenty of time before Batu arrives.`
```

			choice
				`	(Take the direct route.)`
					goto "take the direct route"
				`	(Enjoy the sights along the way.)`

> 			`	Moscow is thousands of kilometers away, so you travel most of the route on Shuttles. It seems some new pilots took an option you never considered: planet-side travel in an exclusive contract with local transit authorities who pay half the mortgage. Certainly a safe career, but not as exciting as your experiences.`

Reviewer 2: ```suggestion
			`	Moscow is thousands of kilometers away, so you travel most of the route on shuttlecraft. It seems some new pilots took an option you never considered: planet-side travel in an exclusive contract with local transit authorities who pay half the mortgage. Certainly a safe career, but they'll never experience space like you have.`
```

Author: The change to the last sentence doesn't work. Those pilots may experience space like you have, but not until they can afford a new ship and leave their exclusive contract.

Reviewer 2: ```suggestion
			`	Moscow is thousands of kilometers away, so you travel most of the route on Shuttles. It seems some new pilots took an option you never considered: planet-side travel in an exclusive contract with local transit authorities who pay half the mortgage. Certainly a safe career, but it's too dull to be restrained to just one planet.`
```
I see what you mean. Is this better?

Author: I have applied those changes. I haven't seen any others though.

Reviewer 2: Yeah, you applied the right ones, thanks.

> 			`	Moscow is thousands of kilometers away, so you travel most of the route on Shuttles. It seems some new pilots took an option you never considered: planet-side travel in an exclusive contract with local transit authorities who pay half the mortgage. Certainly a safe career, but not as exciting as your experiences.`

Reviewer 2: ```suggestion
			`	Moscow is thousands of kilometers away, so you travel most of the route on shuttlecraft. It seems some new pilots took an option you never considered: planet-side travel in an exclusive contract with local transit authorities who pay half the mortgage. Certainly a safe career, but it's too dull to be restrained to just one planet.`
```

> 			`	You fly past mountain ranges, lakes, and deserts, and the occasional tundra. Everywhere, you see the gray cities of Earth packed in their ancient locations. Even from kilometers above, the cities' vast megabuildings are an imposing sight. Still there's some farmland, even now that farming has mostly moved off-world, but as you well know, it isn't enough for the many billions of people on Earth.`

Reviewer 2: ```suggestion
			`	You fly past mountain ranges, lakes, and deserts, and the occasional tundra. Everywhere, you see the gray cities of Earth packed in their ancient locations. Even from kilometers above, the cities' vast megabuildings are an imposing sight. There's still some farmland, even now that farming has largely moved off-world; but as you well know, all of that isn't enough for the twenty billion people living on Earth.`
```

> 			`	You fly past mountain ranges, lakes, and deserts, and the occasional tundra. Everywhere, you see the gray cities of Earth packed in their ancient locations. Even from kilometers above, the cities' vast megabuildings are an imposing sight. There's still some farmland, even now that farming has largely moved off-world; but as you well know, all of that isn't enough for the twenty billion people living on Earth.`

Reviewer: Just correcting for the actual geography in between those places.
```suggestion
			`	You fly past mountain ranges, lakes, and tundra, both bare and forested. The gray cities of Earth stand out when you pass them; packed in their ancient locations and their new ones. Even from far above, the cities' vast megabuildings are an imposing sight. There's still some farmland on this route, even now that farming has largely moved off-world; but as you well know, all of that isn't enough for the twenty billion people living on Earth.`
```

Reviewer 2: You would pass over the Karakorum Desert (Kazakhstan) if you landed at Ulaanbaatar. Mongolia is half desert itself.

Reviewer: The Karakorum Desert is in Turkmenistan, not Kazakhstan, and you wouldn't pass over it on the way to Nizhny.
The recognisably desert part of Mongolia is also south of any vaguely direct route you would take from Ulaanbaatar.
While other areas might technically be "deserts", they're also more accurately arid tundras.
So no.

Reviewer 3: I thought the player was going Ulaanbataar -> Moscow, not Nizhny?

Anyway regardless of the exact Karakorum desert positioning, most of Central Asia is desertic/arid (regardless of the exact classification) so it bears mentioning.

Author: You don't know how climate change and terraforming will change Earth between now and then. I was just putting some geographical features in the paragraph; I don't particularly care which ones.  (Other than "ocean" or "dozens of super volcanoes.") If you can convince MasterOfGrey it needs to be something else, I'll change it.

Reviewer 2: If you take 3 shuttles like it says, you'd probably land or go over somewhere around Astana. From there, you would be able to see desert. There's a Karakum desert in Kazakhstan, but that one is covering the Aral Sea.

Author: From another comment chain, @bene-dictator pointed out that Tundras don't have forests (or to put it another way, if it has a forest, it is not a tundra).

I researched it and found this: "Tundra comes from the Finnish word tunturi, meaning treeless plain." There are a lot of informational sites on the flora and fauna of tundras, and although there may be scattered trees, the foliage is mostly quite low.

So even the word itself says it is treeless. No forested tundras, sorry.

			`	After three shuttle trips, you land in the city of Nizhny, and board a two-story train that hovers in a tunnel of superconducting magnets. Such a complex system would be unprofitable anywhere else in human space, but the population is so high here that these village-sized trains run non-stop at full capacity.`
			choice
				`	(Exit the train at Moscow.)`
			`	Strolling through Moscow to your destination, you pass by many historic sites. Saint Basil’s Cathedral catches your eye, an ancient religious site that Ivan the Terrible ordered to be built. That was almost one and a half millennia ago, but it has been painstakingly preserved all this time. It is shadowed on three sides by monolithic buildings larger than any ship you've seen, but there is a large array of flood lights keeping the cathedral lit.`

> 			`	You notice the building behind the cathedral is your destination. After a brief walk, you pass through its main doors into a room as big as football field, filled with in-building trains, moving walkways, and hover platform elevators. You find an information desk, and inquire the meaning of 'room number: 3017A19' - the 301st floor, section 7A, room 19 - and get directions. It's amazing how large Earth's megabuildings have become. Some planets have smaller populations than the building you are standing in right now, and it is but one of hundreds you've seen on your short journey.`

Reviewer 2: ```suggestion
			`	You notice the building behind the cathedral is your destination. After a brief walk, you pass through its main doors into a room as big as a football field, filled with in-building trains, moving walkways, and hover-platform elevators. You find an information desk, and inquire about the meaning of 'room number: 3017A19' - the 301st floor, section 7A, room 19 - and get directions. It's amazing how large Earth's megabuildings have become; some planets have smaller populations than the building you are standing in right now, and it is but one of hundreds you've seen on your short journey.`
```

> 			`	You notice the building behind the cathedral is your destination. After a brief walk, you pass through its main doors into a room as big as a football field, filled with in-building trains, moving walkways, and hover-platform elevators. You find an information desk, and inquire about the meaning of 'room number: 3017A19' - the 301st floor, section 7A, room 19 - and get directions. It's amazing how large Earth's megabuildings have become; some planets have smaller populations than the building you are standing in right now, and it is but one of hundreds you've seen on your short journey.`

Reviewer: Hard no on it being the building next to a major landmark
```suggestion
			`	At length you arrive at the building you seek and pass through its main doors into a room as big as a football field, filled with in-building trains, moving walkways, and hover-platform elevators. You find an information desk, and inquire about the meaning of 'room number: 3017A19' - the 301st floor, section 7A, room 19 - and get directions. It's amazing how large Earth's megabuildings have become; some planets have smaller populations than the building you are standing in right now, and it is but one of hundreds you've seen on your short journey.`
```

			choice
				`	(Head to the 301st floor, section 7A, room 19.)`
			`	Reaching the room in question you knock on the door which has a nameplate that reads, 'Giti Gilani' with the title 'Research Librarian' in small print underneath. You are swiftly greeted by a short, energetic looking lady. She looks you up and down and nods. "Captain <last>, you're not as tall as I thought you'd be in person. Please, please come in."`
				goto "converse with giti"
			label "take the direct route"

> 			`	You're ushered to a luxury hover-taxi with built-in bar and entertainment. It moves at hypersonic speeds, reaching your destination in under twenty minutes. The taxi attaches its aft exit to the side of the building, about half the way up. The well-dressed driver leads you into the building and escorts to your destination.`

Reviewer 2: ```suggestion
			`	You're ushered into a luxury hover-taxi, complete with built-in bar and the latest forms of entertainment. It moves at hypersonic speeds, reaching your destination in under twenty minutes. The taxi attaches its aft exit to the side of the building, about half the way up. The well-dressed driver leads you into the building and escorts to your destination.`
```
Is the taxi going through space? Hypersonic speeds over Earth can be dangerous for the people on the ground (glass and popped eardrums). I also don't know how 31st century entertainment looks like but I'll just make it vague and let the player decide what it means.

Author: There are experiments today into making supersonic jets that produce almost no boom. It's probably best to remove those details from the paragraph though, plus this is the dialog for players that want to skip the wordy text. I'll update this shortly.

Author: It is no longer a hover taxi. Instead, it enters a ballistic orbit into space. I made your other changes (into, complete, the latest forms of) manually.

> 			`	You're ushered into a luxury taxi, complete with built-in bar and the latest forms entertainment. The taxi enters a ballistic trajectory into space and back down, reaching your destination in under an hour. It links its aft exit port to the side of the building, about half the way up. The well-dressed driver leads you into the building and escorts to your destination.`

Reviewer 2: ```suggestion
			`	You're ushered into a luxury taxi, complete with a built-in bar and the latest forms of entertainment. The taxi enters a ballistic trajectory into space and back down, reaching your destination in under an hour. It links its aft exit port to the side of the building, about half the way up. The well-dressed driver leads you into the building and escorts to your destination.`
```

> 			`	You're ushered into a luxury taxi, complete with a built-in bar and the latest forms of entertainment. The taxi enters a ballistic trajectory into space and back down, reaching your destination in under an hour. It links its aft exit port to the side of the building, about half the way up. The well-dressed driver leads you into the building and escorts to your destination.`

Reviewer: ```suggestion
			`	You're ushered into a luxury taxi, complete with a built-in bar and the latest forms of entertainment. The taxi enters a ballistic trajectory into space and back down, reaching your destination in under an hour. It links its aft exit port to a hover vehicle access port about half the way up the side of the building. The well-dressed driver leads you into the building and, after briefly consulting a directory, escorts you to your destination.`
```

		log "Giti identified the angry cat symbol. It is from a long-defunct subversive group within the Syndicate called Base Principles. Somehow they obtained alien technology, and used the knowledge gained from it to advance their influence within the organization. A half-century ago, Syndicate Internal Affairs scoured them from the Syndicate's ranks in an extremely thorough and prejudiced inquisition."
		
		conversation
			`You land at Ulaanbaatar's spaceport and receive a message that Batu has to finalize some arrangements and will arrive in the evening. That leaves plenty of time for you to see to your other business about the symbol.`

> 			`	At an information desk, you inquire about the route to Moscow, and are given two options. You can take a very expensive private taxi service, which you could bill to Alondo. Or, you can take the long route and see the sights. Either way, you'll have plenty of time before Batu arrives.`

Reviewer 2: ```suggestion
			`	At an information desk, you inquire about the route to Moscow, and are given two options. You can take a very expensive private taxi service, which you could bill to Alondo. Or, you can travel by shuttle, which is slower but offers the chance to see some of the sights of Earth. Either way, you'll have plenty of time before Batu arrives.`
```

Author: The long route also involves a train and some walking. "By shuttle" implies that the shuttle takes you all the way, which is actually the other option.

Reviewer 2: Okay, but the information desk will tell you to take some flights to Nizhny Novgorod first. I'm pretty sure that's not mentioned until you get there. 
```suggestion
			`	At an information desk, you inquire about the route to Moscow, and are given two options. You can take a very expensive private taxi service, which you could bill to Alondo. Or, you can travel by shuttle to Nizhny and take the train to Moscow from there. It'll be slower, but you'll have the chance to see some of the sights of Earth. Either way, you'll have plenty of time before Batu arrives.`
```

Reviewer 2: Nevermind, it wouldn't work with the next bit.

> 			`	At an information desk, you inquire about the route to Moscow, and are given two options. You can take a very expensive private taxi service, which you could bill to Alondo. Or, you can take the long route and see the sights. Either way, you'll have plenty of time before Batu arrives.`

Reviewer 2: ```suggestion
			`	At an information desk, you inquire about the route to Moscow, and are given two options. You can take a very expensive private taxi service, which you could bill to Alondo. Or, you can take a longer route and see the sights. Either way, you'll have plenty of time before Batu arrives.`
```

Author: "The long route" is a figure of speech here. The actual distance is probably about the same.

Reviewer 2: I didn't realise that. I just thought that having to go by 3 shuttles, the player would have to take a more round-about route.

Author: The taxi goes out to space and back down, but it spends most of the time in space, so the duration is shorter, but the distance traveled is longer.

We're splitting hairs, though. The intent is to use the figure of speech, not a quantification of the relative distance.

> 			`	She information desk staff asks you, "Should I contact the direct taxi service, or would you rather see the sights along the way?"`

Reviewer 2: ```suggestion
			`	The attendant at the information desk asks you, "Would you like to contact the direct taxi service, or would you rather see the sights along the way?"`
```

> 			`	The attendant at the information desk asks you, "Would you like to contact the direct taxi service, or would you rather see the sights along the way?"`

Reviewer 2: ```suggestion
			`	The attendant at the information desk asks you, "Would you like to contact the direct taxi service, or take the slow route and see the sights?"`
```
I'm going to change the last paragraph to make this change fit a little better.

			choice
				`	"I'm in a hurry. I should take the direct route."`
					goto "take the direct route"
				`	"The slow route sounds more scenic. I'll try that."`

> 			`	Moscow is thousands of kilometers away, so you travel most of the route on shuttlecraft. It seems some new pilots took an option you never considered: planet-side travel in an exclusive contract with local transit authorities who pay half the mortgage. Certainly a safe career, but it's too dull to be restrained to just one planet.`

Reviewer 2: ```suggestion
			`	Moscow is thousands of kilometers away, so you travel most of the route on shuttlecraft. It seems some new pilots took an option you never considered: planet-side travel in an exclusive contract with local transit authorities who pay half the mortgage. Certainly a safe career, but it would be too dull to be restrained to just one planet.`
```

> 			`	You fly past mountain ranges, lakes, and tundra, both bare and forested. The gray cities of Earth stand out when you pass them; packed in their ancient locations and their new ones. Even from far above, the cities' vast megabuildings are an imposing sight. There's still some farmland on this route, even now that farming has largely moved off-world; but as you well know, all of that isn't enough for the twenty billion people living on Earth.`

Reviewer 2: ```suggestion
			`	You fly past mountain ranges and lakes, and over bare tundras and steppes. The gray cities of Earth stand out when you pass them; packed in their ancient locations and their new ones. Even from far above, the cities' vast megabuildings are an imposing sight. There's still some farmland on this route, even now that farming has largely moved off-world; but as you well know, all of that isn't enough for the twenty billion people living on Earth.`
```
Tundra isn't forested. Even if you manage to miss the desert, you'll see the steppes.

Author: Found this while researching the subject, "Tundra comes from the Finnish word tunturi, meaning *treeless* plain."

> 			`	You fly past mountain ranges and lakes, and over bare tundras and steppes. The gray cities of Earth stand out when you pass them; packed in their ancient locations and their new ones. Even from far above, the cities' vast megabuildings are an imposing sight. There's still some farmland on this route, even now that farming has largely moved off-world; but as you well know, all of that isn't enough for the twenty billion people living on Earth.`

Reviewer 4: ```suggestion
			`	You fly past mountain ranges and lakes, and over bare tundras and steppes. The gray cities of Earth stand out when you pass them; packed in their ancient locations and their new ones. Even from far above, the cities' vast megabuildings are an imposing sight. There's still some farmland on this route, even now that farming has largely moved off-world, because, as you well know, all of that isn't enough for the twenty billion people living on Earth.`
```

This feels a lot smoother a way to split this up, but I'm not entirely sure it's appropriate anyway, as an assumption of what player "well knows."

Author: I changed it, so it doesn't tell the player that they understand the food economics of the Republic. (No more "as you well know.")

> 			`	After three shuttle trips, you land in the city of Nizhny and board a two-story train that hovers in a tunnel of superconducting magnets. Such a complex system would be unprofitable anywhere else in human space, but the population is so high here that these village-sized trains run non-stop at full capacity. As you look lost in the vast station, a passerby asks if you need directions.`

Reviewer: ```suggestion
			`	After three shuttle trips, you land in the city of Nizhny and board a two-story train that hovers in a tunnel of superconducting magnets. Such a complex system would be unprofitable anywhere else in human space, but the population is so high here that these village-sized trains run non-stop at full capacity. You must look lost as you disembark in the vast station in Moscow, and a passerby asks if you need directions.`
```

> 			`	After three shuttle trips, you land in the city of Nizhny, and board a two-story train that hovers in a tunnel of superconducting magnets. Such a complex system would be unprofitable anywhere else in human space, but the population is so high here that these village-sized trains run non-stop at full capacity. As you look lost in the vast station, a passerby asks if you need directions.`

Reviewer 4: ```suggestion
			`	After three shuttle trips, you land in the city of Nizhny and board a two-story train that hovers in a tunnel of superconducting magnets. Such a complex system would be unprofitable anywhere else in human space, but the population is so high here that these village-sized trains run non-stop at full capacity. As you look lost in the vast station, a passerby asks if you need directions.`
```

			choice
				`	"How do I get to the surface?"`
				`	"I'm looking for this address."`

> 			`	You follow the directions, strolling through Moscow to your destination, you pass by many historic sites. Saint Basil’s Cathedral catches your eye, an ancient religious site that Ivan the Terrible ordered to be built. That was almost one and a half millennia ago, but it has been painstakingly preserved all this time. It is shadowed on three sides by monolithic buildings larger than any ship you've seen, but there is a large array of flood lights keeping the cathedral lit.`

Reviewer 2: ```suggestion
			`	You follow his directions, and while strolling through Moscow to your destination, you pass by many historic sites. Saint Basil’s Cathedral catches your eye, an ancient religious site that Ivan the Terrible ordered to be built. That was almost one and a half millennia ago, but it has been painstakingly preserved all this time. It is shadowed on three sides by monolithic buildings larger than any ship you've seen, but there is a large array of flood lights keeping the cathedral lit.`
```

> 			`	You follow his directions, and while strolling through Moscow to your destination, you pass by many historic sites. Saint Basil’s Cathedral catches your eye, an ancient religious site that Ivan the Terrible ordered to be built. That was almost one and a half millennia ago, but it has been painstakingly preserved all this time. It is shadowed on three sides by monolithic buildings larger than any ship you've seen, but there is a large array of flood lights keeping the cathedral lit.`

Reviewer 4: ```suggestion
			`	You follow his directions and, while strolling through Moscow to your destination, pass many historic sites. Saint Basil’s Cathedral, an ancient religious site that Ivan the Terrible ordered to be built, catches your eye. That was almost one and a half millennia ago, but it has been painstakingly preserved all this time. It is shadowed on three sides by monolithic buildings larger than any ship you've seen, but there is a large array of flood lights keeping the cathedral lit.`
```

> 			`	At length you arrive at the building you seek and pass through its main doors into a room as big as a football field, filled with in-building trains, moving walkways, and hover-platform elevators. It's amazing how large Earth's megabuildings have become; some planets have smaller populations than the building you are standing in right now, and it is but one of hundreds you've seen on your short journey.`

Reviewer 2: ```suggestion
			`	At last you arrive at the building you're looking for. Passing through its main doors, you find youself in a room as big as a football field, filled with in-building trains, moving walkways, and hover-platform elevators. It's amazing how large Earth's megabuildings have become; some planets have smaller populations than the building you are standing in right now, and it is but one of hundreds you've seen on your short journey.`
```

> 			`	At length you arrive at the building you seek and pass through its main doors into a room as big as a football field, filled with in-building trains, moving walkways, and hover-platform elevators. It's amazing how large Earth's megabuildings have become; some planets have smaller populations than the building you are standing in right now, and it is but one of hundreds you've seen on your short journey.`

Reviewer 2: ```suggestion
			`	At length you arrive at the building you seek, and pass through its main doors into a room as big as a football field, filled with in-building trains, moving walkways, and hover-platform elevators. It's amazing how large Earth's megabuildings have become; some planets have smaller populations than the building you are standing in right now, and it is but one of hundreds you've seen on your short journey.`
```

> 			`	At length you arrive at the building you seek, and pass through its main doors into a room as big as a football field, filled with in-building trains, moving walkways, and hover-platform elevators. It's amazing how large Earth's megabuildings have become; some planets have smaller populations than the building you are standing in right now, and it is but one of hundreds you've seen on your short journey.`

Reviewer 4: ```suggestion
			`	At length you arrive at the building you seek and pass through its main doors into a room as big as a football field, filled with in-building trains, moving walkways, and hover-platform elevators. It's amazing how large Earth's megabuildings have become; some planets have smaller populations than the building you are standing in right now, and it is but one of hundreds you've seen on your short journey.`
```

			`	You find an information desk, and inquire about the meaning of 'room number: 3017A19' - the 301st floor, section 7A, room 19 - and get directions. They point you to what looks like a cross between a train and an elevator. A robotic voice asks your destination.`
			choice
				`	"Please take me to the 301st floor, section 7A, room 19."`
				`	"Room 3017A19, please."`
			`	Reaching the room in question you knock on the door which has a nameplate that reads, 'Giti Gilani' with the title 'Research Librarian' in small print underneath. You are swiftly greeted by a short, energetic looking lady. She looks you up and down and nods. "Captain <last>, you're not as tall as I thought you'd be in person. Please, please come in."`
				goto "converse with giti"
			label "take the direct route"

> 			`	You're ushered into a luxury taxi, complete with a built-in bar and the latest forms of entertainment. The taxi enters a ballistic trajectory into space and back down, reaching your destination in under an hour. It links its aft exit port to a hover vehicle access port about half the way up the side of the building. The well-dressed driver leads you into the building and, after briefly consulting a directory, escorts you to your destination.`

Reviewer 5: ```suggestion
			`	You're ushered into a luxury taxi, complete with a built-in bar and the latest forms of entertainment. The taxi enters a ballistic trajectory into space and back down into the atmosphere, reaching your destination in under an hour. It links its aft exit port to a hover vehicle access port about half the way up the side of the building. The well-dressed driver leads you into the building and, after briefly consulting a directory, escorts you to your destination.`
```

			`	She frowns at you. "No. Somewhat more recently, only a few decades back in fact, the same symbol came up again in conjunction with a smuggling ring that was kidnapping gifted children. It seems that a minor error resulted in a crate being misplaced by the smugglers, complete with a child inside and the symbol surreptitiously stamped on the corner plate. The ship the crate was meant to be on self-destructed when it was caught, and any potential affiliates went to ground. A handful of ships and individuals were identified retrospectively to have been involved, but none of them were ever seen again. The kidnappings stopped."`
			label newpictures
			`	She waves a hand dismissively at her screen. "Of course, I only received the one image to refer to, and the older records are not necessarily very good quality or captured with future research librarians in mind. So, please, show me what else you have."`
			choice
				`	(Show her new pictures from your datapad.)`
			`	You pull out your datapad and hand it to her with the folder of relevant pictures, some carefully redacted of Hai paraphernalia, for her to look at. As she begins flicking through them she makes some comments.`

> 			`	"You know, there's quite a lot more of these than I expected. All quite recent too judging by the file quality and that crate design. They are exactly the same as the original. Someone clearly has made a deliberate decision to reuse the exact symbol, no re-imagining or stylistic alterations whatsoever. If you ask me that sounds like a group that has set itself to having the same goals, which means the Syndicate would probably be very interested in this. Though I suppose you'd have the connections there if... Oh!" She pauses. "Now this is interesting."`

Reviewer 2: ```suggestion
			`	"You know, there's quite a lot more of these than I expected. All quite recent too judging by the file quality and that crate design. They are exactly the same as the original. Someone clearly has made a deliberate decision to reuse the exact symbol, no re-imagining or stylistic alterations whatsoever. If you ask me, that sounds like a group that has set itself to having the same goals, which means the Syndicate would probably be very interested in this. Though I suppose you'd have the connections there if... Oh!" She pauses. "Now this is interesting."`
```

			`	She turns the datapad to you and shows you what looks like an amulet or maybe a jewelry charm. The scale in the image isn't very clear. "Do you see this patterning around the outside of the symbol on this? I'm quite sure nearly everyone in the galaxy would simply pass that off as decorative, but it's not.`
			choice

> 				`	(Examine the symbols.)`

Reviewer: This might be one break-up more than we really need here.

Author: I need to revisit "Examine the symbols" since Amazinite wants me to remove any "action choices."

Author: When I take it out, it becomes a text wall. However, I replaced "Examine the Symbols" with a more meaningful choice where you can impress Giti with your insight.

I just made major changes to the conversation to remove all the action choices. You should re-read it in its entirety.

			`	"You know, there's quite a lot more of these than I expected. All quite recent too judging by the file quality and that crate design. They are exactly the same as the original. Someone clearly has made a deliberate decision to reuse the exact symbol, no re-imagining or stylistic alterations whatsoever. If you ask me, that sounds like a group that has set itself to having the same goals, which means the Syndicate would probably be very interested in this. Though I suppose you'd have the connections there if... Oh!" She pauses. "Now this is interesting."`
			`	She turns the datapad to you and shows you what looks like an amulet or maybe a jewelry charm. The scale in the image isn't very clear. "Do you see this patterning around the outside of the symbol on this? I'm quite sure nearly everyone in the galaxy would simply pass that off as decorative, but it's not." She pauses for a moment, perhaps expecting a response.`
			choice
				`	(Politely wait for her to finish.)`
					goto disappointed

> 				`	"I have a hunch you're going to tell it's a form of writing."`

Reviewer 4: ```suggestion
				`	"I have a hunch you're going to tell me it's a form of writing."`
```

			`	She looks impressed and says, "Very good! I underestimated you, Captain <last>.`
				goto archaic
			label disappointed

> 			`	After a couple seconds, Giti looks slightly disappointed, as if you failed an intelligence test.`

Reviewer: We're not usually this explicit that the player made the "wrong" choice.
```suggestion
			`	After a couple seconds, Giti looks slightly disappointed but makes no comment on the matter.`
```

			label newpictures
			`	She waves a hand dismissively at her screen. "Of course, I only received the one image to refer to, and the older records are not necessarily very good quality or captured with future research librarians in mind. So, please, show me what else you have."`
			choice
				`	"I have some pictures to show you."`
				`	"Can you glean more from this?"`
			`	You pull out your datapad and hand it to her with the folder of relevant pictures, some carefully redacted of Hai paraphernalia, for her to look at. As she begins flicking through them she makes some comments.`

> 			`	"You know, there's quite a lot more of these than I expected. All quite recent too judging by the file quality and that crate design. They are exactly the same as the original. Someone clearly has made a deliberate decision to reuse the exact symbol, no re-imagining or stylistic alterations whatsoever. If you ask me, that sounds like a group that has set itself to having the same goals, which means the Syndicate would probably be very interested in this. Though I suppose you'd have the connections there if... Oh!" She pauses. "Now this is interesting."`

Reviewer 5: ```suggestion
			`	"You know, there's quite a lot more of these than I expected. All quite recent too, judging by the file quality and that crate design. They are exactly the same as the original. Someone clearly has made a deliberate decision to reuse the exact symbol, with no re-imagining or stylistic alterations whatsoever. If you ask me, that sounds like a group that has set itself to having the same goals, which means the Syndicate would probably be very interested in this. Though I suppose you'd have the connections there if... Oh!" She pauses. "Now this is interesting."`
```

Author: The problem with adding commas is that someone else inevitably asks me to remove the same comma a few says later. I committed your suggestion anyway. I'll probably commit theirs too. It is getting to the point where I'm wondering if I even know what a comma is anymore, let alone what they're for (,,,,,).
